File: EduChemSuite.API/EduChemSuite.API/Models/DistrictSchoolsModel.cs
namespace EduChemSuite.API.Models;

public class DistrictSchoolsModel
{
        public Guid DistrictId { get; set; }
        public virtual DistrictModel? District { get; set; }
        public Guid SchoolId { get; set; }
        public virtual SchoolModel? School { get; set; }
}

File: EduChemSuite.UI/UI/src/app/_models/DistrictSchoolsModel.ts
import {UserModel} from "./UserModel";
import {DistrictModel} from "./DistrictModel";
import {SchoolModel} from "./SchoolModel";

export class DistrictSchoolsModel{
  schoolId?: string;
  school?: SchoolModel;
  districtId?: string;
  district?: DistrictModel;
}
